main.tf

```diff
@@ -44,7 +44,14 @@ resource "aws_security_group_rule" "egress" {
   security_group_id = join("", aws_security_group.default.*.id)
 }
 
-resource "aws_rds_cluster" "primary" {
```

Comment:
Changing the resource name will destroy every single cluster already created using this module, this is a not needed breaking change.